models/src/main/scala/coop/rchain/models/SortedParMap.scala

```diff
@@ -44,15 +44,25 @@ final case class SortedParMap(ps: TreeMap[Par, Par]) extends Iterable[(Par, Par)
 }
 
 object SortedParMap {
+  @SuppressWarnings(Array("org.wartremover.warts.Var", "org.wartremover.warts.Null"))
```

Comment:
@ArturGajowy @KentShikama This is why I used `TreeMap[ScoredTerm[Par], Par]`. No cache, no resorting.